js/Data/State_indicators.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Enumerator representing all 50 States and DC
+ */
+export enum State_id 
```

Comment:
Can the MigrationNodeId type in MigrationsPatterns.ts be used here instead of creating a new type?